lib/Dashboard/CalendarWidget.php

```diff
@@ -124,6 +125,9 @@ public function getItems(string $userId, ?string $since = null, int $limit = 7):
 		];
 		$widgetItems = [];
 		foreach ($calendars as $calendar) {
+			if ($calendar instanceof ICalendarIsEnabled && $calendar->isEnabled() === false) {
```

Comment:
Needs to be guarded by `if (interface_exists(ICalendarIsEnabled) ...)` until we are sure that this interface is always available.